pkgs/by-name/mi/microfetch/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "microfetch";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "NotAShelf";
+    repo = "microfetch";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

I'm not a huge fan of `rec` but this'll do.

---

Applied, I can replace `rec` by `let in` if that's what you had in mind.

---

It was, but thats a whole debate on its own. I'm not going to push in either direction, so use what you prefer.